build.ps1

```diff
@@ -67,8 +67,20 @@ function buildSolution{
 function executeTests{
 
     Write-Host "Execute Tests"
-    $nunitConsole = "$rootFolder\packages\NUnit.Runners.2.6.3\tools\nunit-console.exe"
-    & $nunitConsole .\Source\Loggly.Tests\bin\$configuration\Loggly.Tests.dll
+
+    $testResultformat = ""
+    $nunitConsole = "$rootFolder\packages\NUnit.ConsoleRunner.3.4.1\tools\nunit3-console.exe"
+
+    if(Test-Path Env:\APPVEYOR){
+        $testResultformat = ";format=AppVeyor"
+        $nunitConsole = "nunit3-console"
```

Comment:
AppVeyor seems to have a customized runner that report test status in real-time to their service.